genesyscloud/util/util_retries.go

```diff
@@ -37,6 +37,7 @@ func WithRetriesForReadCustomTimeout(ctx context.Context, timeout time.Duration,
 		if strings.Contains(fmt.Sprintf("%v", err), "API Error: 404") {
 			// Set ID empty if the object isn't found after the specified timeout
 			d.SetId("")
+			return nil
```

Comment:
Ultimately, this is where the true fix for the issue lives. We don't throw an error if the resource cannot be found. Instead, we just set the ID to empty after the timeout and immediately return.